src/components/guide/tvguide.template.html

```diff
@@ -9,8 +9,8 @@
 
     <div class="guide-headerTimeslots">
         <div class="guide-channelTimeslotHeader">
-            <button is="paper-icon-button-light" type="button" class="btnGuideViewSettings">
-                <span class="material-icons btnGuideViewSettingsIcon more_horiz"></span>
+            <button is="paper-icon-button-light" type="button" class="btnGuideViewSettings" title="${ButtonMore}">
+                <span class="material-icons btnGuideViewSettingsIcon more_horiz" aria-hidden="true"></span>
```

Comment:
_Not a blocker_
This button opens dialog "Settings". Maybe we should name it as dialog that opens and change icon.

Reply:
I don't have access to the TV part of the client, so I'm leaving this for someone else :)

Reply:
I added free M3U provider. No guide, but some LiveTV features can be tested.